stdlib/public/Darwin/Accelerate/vForce_Operations.swift

```diff
@@ -0,0 +1,3435 @@
+    @inline(__always)
+    @available(iOS 9999, macOS 9999, tvOS 9999, watchOS 9999, *)
+    public static func nearestInteger<U, V>(_ vector: U,
+                                            result: inout V)
+        where
+        U : _ContiguousCollection,
+        V : _MutableContiguousCollection,
+        U.Element == Float, V.Element == Float {
+            
+            precondition(vector.count == result.count)
+            
+            var n = Int32(vector.count)
+            
+            result.withUnsafeMutableBufferPointer { dest in
+                vector.withUnsafeBufferPointer { src in
+                    vvnintf(dest.baseAddress!,
+                            src.baseAddress!,
+                            &n)
+                }
+            }
+    }
+    
+    /// Returns the nearest integer to each element in `vector`, double-precision.
+    ///
+    /// - Parameter _ vector: Input values.
+    /// - Returns: Output values.
+    @inline(__always)
+    @available(iOS 9999, macOS 9999, tvOS 9999, watchOS 9999, *)
+    public static func nearestInteger<U>(_ vector: U)  -> [Double]
+        where
+        U : _ContiguousCollection,
+        U.Element == Double {
+            
+            let result = Array<Double>(unsafeUninitializedCapacity: vector.count) {
+                buffer, initializedCount in
+                
+                nearestInteger(vector,
+                               result: &buffer)
+                
+                initializedCount = vector.count
+            }
+            
+            return result
+    }
+    
+    /// Calculates the nearest integer to each element in `vector`, writing the result to `result`, double-precision.
+    ///
+    /// - Parameter _ vector: Input values.
+    /// - Parameter result: Output values.
+    @inline(__always)
+    @available(iOS 9999, macOS 9999, tvOS 9999, watchOS 9999, *)
+    public static func nearestInteger<U, V>(_ vector: U,
+                                            result: inout V)
+        where
+        U : _ContiguousCollection,
+        V : _MutableContiguousCollection,
+        U.Element == Double, V.Element == Double {
+            
+            precondition(vector.count == result.count)
+            
+            var n = Int32(vector.count)
+            
+            result.withUnsafeMutableBufferPointer { dest in
+                vector.withUnsafeBufferPointer { src in
+                    vvnint(dest.baseAddress!,
+                           src.baseAddress!,
+                           &n)
+                }
+            }
+    }
+    
+    // MARK: Reciprocal Square Root
+    
+    /// Returns the reciprocal square root of each element in `vector`, single-precision.
+    ///
+    /// - Parameter _ vector: Input values.
+    /// - Returns: Output values.
+    @inline(__always)
+    @available(iOS 9999, macOS 9999, tvOS 9999, watchOS 9999, *)
+    public static func reciprocalSquareRoot<U>(_ vector: U)  -> [Float]
```

Comment:
I'm torn between `rsqrt` and `reciprocalSquareRoot` here. I think `rsqrt` may be OK, but if you want to stick with the current verbose spelling, that's also OK with me.

Reply:
If we're changing `squareRoot` to `sqrt`, I think we should use `rsqrt` for consistency.